torch/library.h

```diff
@@ -952,6 +952,18 @@ class TorchLibraryInit final {
 
 } // namespace detail
 
+// ATenFallbackControlImpl enables backends to explictly control CPU fallback for any op
```

Comment:
This relies on the m.fallback() implementation of CPU fallback. Which doesn't live here. So this is a bit surprising?